### ISSUE TITLE
Element "Información Adicional"

### ISSUE THREAD

**Issue by ghost** [SIB-Colombia/mamut#53]
- The URL of "Enlace multimedia" must show too when you save. Only shows the Source (Fuente) for now.
- The API for Flickr is the only one that retrieve information. When we have the information of Youtube, xeno-canto through the API?

**Comment by mcubillos** [comment]
Fixed in 89849b4ca4ad82e5e4d8cb2e3ed25d8b9e8d9307 and in this moment is working Flickr, Wikicommons and Youtube (with some fields)

**Comment by ghost** [comment]
To clarify some metadata elements callback from **Youtube** I share this [video](https://www.youtube.com/watch?v=DhIgCcG-3_4) with Creative Commons License

**Comment by ghost** [comment]
The correct structure to make the callback to **Wikicommons** API is like this examples:
https://commons.wikimedia.org/wiki/File:2009-03-29Dendrobates_auratus110.jpg
https://commons.wikimedia.org/wiki/File:Saguinus_leucopus.JPG
https://commons.wikimedia.org/wiki/File:Ara_macao_-Fort_Worth_Zoo-8.jpg

**Comment by mcubillos** [comment]
This is the result of calling YouTube API with the video with CC
[callback](https://files.zenhub.io/56d716b2505047d8147a176f)
Uploaded using [**ZenHub.io**](http://www.zenhub.io)

**Comment by mcubillos** [comment]
Yes, that is the correct way to put the url, because the app search 'commons.wikimedia.org' into the url and get the file name of the image after ':'.

**Comment by ghost** [comment]
What happened with xeno-canto callback?

**Comment by mcubillos** [comment]
Xeno-canto API doesn't work correctly in this moment the API returns  'Uncaught SyntaxError: Unexpected token :' error when mamut tries to get the callback information.
